src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/QuickAccessParametersTest.kt

```diff
@@ -0,0 +1,270 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.uiTest
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.fixtures.JLabelFixture
+import com.intellij.remoterobot.fixtures.JListFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.customfixtures.BasicArrowButtonFixture
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.customfixtures.JSpinnerFixture
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages.IdeaFrame
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages.QuickAccessParametersFixtures
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages.WelcomeFrame
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.utils.RemoteRobotExtension
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestMethodOrder
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.time.Duration
+import java.util.stream.Stream
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(RemoteRobotExtension::class)
+class QuickAccessParametersTest {
+    // Open the tool window frame
+    private lateinit var quickAccessParameters: QuickAccessParametersFixtures
```

Comment:
what does a `lateinit` do?

Reply:
It tells the compiler that the variable will be initialised later